FrontDesk/tests/UnitTests/Core/AggregatesEntities/AppointmentTests/Appointment_Create.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using AutoFixture;
+using FrontDesk.Core.Aggregates;
+using Xunit;
+
+namespace UnitTests.Core.AggregatesEntities.AppointmentTests
+{
+  public class Appointment_Create
+  {
+    private Fixture _fixture = new Fixture();
+
+    [Fact]
+    public void CreateSuccess()
+    {
+      var scheduleId = Guid.NewGuid();
+      var appointment = Appointment.Create(scheduleId, 1, 1, 1, DateTime.Now, DateTime.Now.AddHours(3), 1, 1, "Title Test");
+
+      Assert.Null(appointment.DateTimeConfirmed);
+      Assert.Equal(1, appointment.AppointmentTypeId);
+      Assert.Equal(scheduleId, appointment.ScheduleId);
+      Assert.Equal(1, appointment.ClientId);
+      Assert.Equal(1, appointment.DoctorId);
+      Assert.Equal(1, appointment.PatientId);
+      Assert.Equal(1, appointment.RoomId);
+      Assert.Equal(3 * 60, appointment.TimeRange.DurationInMinutes());
+      Assert.Equal("Title Test", appointment.Title);
+    }
+
+    [Fact]
+    public void ThrowExceptionZeroClientId()
+    {
+      var scheduleId = Guid.NewGuid();
+
+      Assert.Throws<ArgumentException>(() =>
+        Appointment.Create(scheduleId, 0, 1, 1, DateTime.Now, DateTime.Now.AddHours(3), 1, 1, "Title Test"));
```

Comment:
Hello, I know this is a WIP but if I can add my 2 cents, I think those unit test would be cleaner if used with the 3A (Arrange, Act and Assert) and separating the actual action from the assertion.

```
// Arrange
var scheduleId = Guid.NewGuid();

// Act
Action act = () => Appointment.Create(scheduleId, 0, 1, 1, DateTime.Now, DateTime.Now.AddHours(3), 1, 1, "Title Test"));

// Assert
Assert.Throws<ArgumentException>(act);
```

Reply:
Interesting approach - I've not seen the action set aside as its own variable like that before in a `Throws` test. I'm not a fan (outside of tutorials) of putting comments for // Arrange // Act // Assert in every test just because that makes them needlessly verbose, but I do try to follow the pattern (with spaces between groups).

Let us get this closer to "done" and maybe you can make a PR for me to consider? If you want to add an issue now with the details so they're not lost when this PR is closed that could be good. thanks!

Reply:
The comments where just to be clear but I find them useful in some cases where it's not so clear. I can open the issue tomorrow and follow up with a PR when this is "done".